src/components/APIDoc/Operation.tsx

```diff
@@ -47,22 +48,13 @@ export const Operation: React.FunctionComponent<OperationProps> = props => {
   </AccordionItem>;
 };
 
+
 const OperationContent: React.FunctionComponent<OperationProps> = ({verb, path, operation, document}) => {
   const parameters = (operation.parameters || []).map(p => deRef(p, document));
 
-  const [codeSampleLanguage, setCodeSampleLanguage] = useState<SnippetInfoItem>(SnippetItemsArray[0]);
-  const reqData: RequestFormat = useMemo(() => ({
-    method: verb.toUpperCase(),
-    url: "http://example.com"+path,
-    httpVersion: "HTTP/1.1",
-    cookies: [],
-    headers: [{name: "Accept", value: "application/json"}], //TODO use headers as per schema. Default to application/json.
-    queryString: [], //TODO path params?
-    postData: undefined, //TODO body params
-    headersSize: -1,
-    bodySize: -1,
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }), [verb, path, codeSampleLanguage]);
+  const [codeSampleLanguage, setCodeSampleLanguage] = useState<SnippetInfoItem>(SnippetItemsArray[4]);
```

Comment:
Could you move python to be the first element of the array and keep this as `SnippetItemsArray[0]` ?

Reply:
done!